MiniApp/Classes/Extensions/MiniApp+URLRequest.swift

```diff
@@ -0,0 +1,9 @@
+extension URLRequest {
+    static func createURLRequest(url: URL, environment: Environment) -> URLRequest {
+        var urlRequest = URLRequest(url: url)
+        if environment.subscriptionKey.isEmpty {
```

Comment:
Isn't this backwards? It should be setting the subscription key if it's not empty?

Reply:
Yes, you were right. Updated the code now.